core/src/main/java/com/datastax/oss/driver/internal/core/tracker/RequestLogger.java

```diff
@@ -137,7 +139,8 @@ public void onError(
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       Node node,
-      @NonNull String logPrefix) {
+      @NonNull String logPrefix,
+      @NonNull ExecutionInfo executionInfo) {
```

Comment:
Similar to with MultiplexingRequestTracker, this is not marked @NonNull in the interface